modules/openy_gc_auth/src/GCUserAuthorizer.php

```diff
@@ -72,11 +85,23 @@ public function authorizeUser($name, $email, array $extra_data = []) {
         $account->save();
       }
     }
+    // List of roles to redirect user login page.
+    $userRolesArray = [
+      'administrator',
+      'virtual_ymca_editor',
+    ];
+    // Redirecting user login page.
+    foreach ($userRolesArray as $role) {
+      if ($account->hasRole($role)) {
+        $loginUrl = Url::fromRoute('user.login')->toString();
+        $this->messenger->addMessage($this->t('You have to login as real user, since you are an administrator.'));
```

Comment:
Saying this user is an administrator could be a security risk. I'd suggest rewriting this to:

> Please retry your login on this form.

Reply:
Updated message. Please review.